pkg/api/customization/monitor/cluster_graph_action.go

```diff
@@ -69,7 +73,16 @@ func (h *ClusterGraphHandler) QuerySeriesAction(actionName string, action *types
 	var svcName, svcNamespace, svcPort, token string
 	if inputParser.Input.Filters["resourceType"] == "istiocluster" {
 		inputParser.Input.MetricParams["namespace"] = ".*"
-		svcName, svcNamespace, svcPort = monitorutil.IstioPrometheusEndpoint()
+		selector := fields.OneTermEqualSelector("metadata.name", monitorutil.IstioAppName)
+		app, err := h.appGetter.Apps("").List(metav1.ListOptions{FieldSelector: selector.String()})
+		if err != nil {
+			return err
+		}
+
+		if len(app.Items) == 0 {
+			return fmt.Errorf("app %s is not found", monitorutil.IstioAppName)
```

Comment:
I don't understand this change. If I enable istio with reusing cluster monitoring, do I still query metrics from `IstioPrometheusEndpoint`?